zulipterminal/helper.py

```diff
@@ -347,3 +354,20 @@ def match_user(user: Any, text: str) -> bool:
         if keyword.startswith(text.lower()):
             return True
     return False
+
+
+@asynch
+def notify(title: str, text: str) -> None:
+    FNULL = open(os.devnull, 'w')
+    command = ""
+    if WSL:
+        command = ('powershell.exe "New-BurntToastNotification'
+                   ' -Text \'{}\', \'{}\'"'.format(title, text))
+    elif MACOS:
+        command = ("osascript -e 'display notification \"{}\" with title"
+                   " \"{}\"'".format(text, title))
+    elif LINUX:
+        command = 'notify-send {} {}'.format(title, text)
+    if command:
+        res = subprocess.run(shlex.split(command),
```

Comment:
Is the shlex actually required, if we construct the command differently?

Reply:
I think it will be better than manually splitting text and feeding it. We will have to take care of a lot of corner cases (probably use some regex to split nested quotes)

Reply:
You're referring to the quotes in the commands you've written, or substituted text?

Reply:
Substituted text.